docs/logs/extending-the-sdk/MyClassWithRedactionEnumerator.cs

```diff
@@ -0,0 +1,72 @@
+// <copyright file="MyRedactionProcessor.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System.Collections;
+using System.Collections.Generic;
+using System.Text;
+using System.Text.RegularExpressions;
+
+internal class MyClassWithRedactionEnumerator : IReadOnlyList<KeyValuePair<string, object>>
+{
+    private static readonly Regex Rgx = new("\\(?\\d{3}[\\.|\\)]?\\d{3}[\\.|\\-]\\d{4}");
```

Comment:
for this tutorial, i dont think we need to show actual regex! its upto end user to make their own.. we can do simple match like value.Contains("secret"), as it is sufficient to demonstrate the approach.

Reply:
+1

Reply:
Or it the class is renamed to "PhoneNumberRedaction(something)" it sounds reasonable.